### PR TITLE
Milestone 4 Forms

### PR DESCRIPTION
**Pull request by Berabjesus**
## Created a simple form with RoR and bootstrap per the project specification
- Creates new user
- Edit user with an id
- Validates empty inputs

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,13 +9,26 @@ def create
...
-    params.permit(:username, :email, :password_digest)
+  def edit
+    @user = User.find(params[:id])
   end
-  # {"username"=>"sdfs", "email"=>"asd@easdas", "password"=>"[FILTERED]", "authenticity_token"=>"wSniYW/JnuQ770uRAoeDaQ4YmwJiFdZZJJ6TW0taBaAzYNXullZhA/e+bowoXqGDuG2ngyUeFL7IfhmkUAkJ+A=="}
 
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: Please, redirect to `edit_uer_path` if successful

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -8,13 +9,26 @@ def create
...
+    else
+      render 'new'
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: In case of error `render 'edit'`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for the job done so far.
However, there are some missing requirements

1. There are some linter's errors. Kindly fix them.

2. Please, the forms are missing. Please, put the two form in comment into the code so that we can check that part of your work

3. Read the comments left below.

Kindly make the suggested changes and submit a new code review request
Thanks

Mael FOSSO

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (Berabjesus) [reply app/controllers/users_controller.rb]: Fixed. Thank you.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (Berabjesus) [reply app/controllers/users_controller.rb]: FIxed, Thank you.

**Review by Ispirett — APPROVED** [review]
## Status Approved 👍 

Great job with the changes made folks 💯

**Pull request merged**